tools/test/README.md

```diff
@@ -0,0 +1,41 @@
+# Integration test for Contile
+
+This directory contains a simple integration test for the Contile server.
+
+## Installation
+
+1) First, create a virtualenv installation of python
+`python -m venv venv`
+
+This will create a local install of the python. You can then "activate" it by calling
+`sh venv/bin/activate.sh`
```

Comment:
Should this be `source venv/bin/activate`? I didn't seem to have a `venv/bin/activate.sh` file 🤔

Reply:
either works, but yeah, that's the proper term.